### PR TITLE
Setup nightly builds for dashboard

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Setup nightly builds for dashboard using the pipeline from the dashboard repo
and the plumbing from the plumbing repo.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by afrittoli** [comment]
luigi at your service! </p>Here is the image you requested: <a href="https://gcr.io/tekton-releases/dogfoodingko:mario">built image</a>|<a href="http://35.222.249.224/?buildid=73f1aa51-d0f7-4ec8-9c3a-fabef4971c4f&namespace=mario">build logs</a>

**Comment by afrittoli** [comment]
mario at your service! </p>Here is the image you requested: <a href="https://gcr.io/tekton-releases/dogfoodingko:mario">built image</a>|<a href="http://35.222.249.224/?buildid=577bde1e-3d11-43c7-8fff-3f310315116e&namespace=mario">build logs</a>

**Comment by afrittoli** [comment]
tekton at your service! </p>Here is the image you requested: <a href="https://gcr.io/tekton-releases/dogfooding/ko:mario">built image</a>|<a href="http://35.222.249.224/?buildid=1c77eafb-1a9e-4dda-8723-73c3b04d9ab7&namespace=mario">build logs</a>

**Comment by afrittoli** [comment]
luigi at your service! </p>Here is the image you requested: <a href="https://gcr.io/tekton-releases/dogfooding/ko:mario">built image</a>|<a href="http://35.222.249.224/?buildid=7b777671-659c-44ed-a04a-c9ba06db12f9&namespace=mario">build logs</a>

File: tekton/cronjobs/dashboard-nightly-release-cron/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+bases:
+- ../nightly-release-cron-base
+patchesStrategicMerge:
+- cronjob.yaml
+nameSuffix: "-dashboard-nightly-release"
```
Comment (bobcatfish) [inline tekton/cronjobs/dashboard-nightly-release-cron/kustomization.yaml]: ahhh this is where `nightly-cron-trigger-dashboard-nightly-release` comes from!

**Pull request merged**